### PR TITLE
HTTP Stats Performance Changes

### PR DESCRIPTION
**Pull request by jdmcclur**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

################################################################################################

This is a change to not use lamdas or streams. This is a bit of a nitpick, but those are not best for performance.

### CODE REVIEW

**Comment by jdmcclur** [comment]
#build

**Comment by jdmcclur** [comment]
#build

**Comment by jdmcclur** [comment]
#build #libby

**Review by Channyboy — APPROVED** [review]
lgtm

File: dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/HttpServerStatsMonitor.java [hunk 1]
```diff
@@ -305,17 +303,16 @@ private String resolveStatsKey(HttpStatAttributes httpStatAttributes) {
...
+		*/
+		if (responseStatus.isPresent()) {
+			sb.append(";status:" + Integer.toString(responseStatus.get()));
+		}
+		if (httpRoute.isPresent()) {
+			sb.append(";httpRoute:" + httpRoute.get().replace("*", "\\*"));
+		}
+		if (errorType.isPresent()) {
+			sb.append(";errorType:" + errorType.get());
+		}
```
Comment (jhanders34) [inline dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/HttpServerStatsMonitor.java:315]:
```suggestion
		if (responseStatus.isPresent()) {
			sb.append(";status:").append(Integer.toString(responseStatus.get()));
		}
		if (httpRoute.isPresent()) {
			sb.append(";httpRoute:").append(httpRoute.get().replace("*", "\\*"));
		}
		if (errorType.isPresent()) {
			sb.append(";errorType:").append(errorType.get());
		}
```

**Pull request merged**